report_page_orientation/README.rst

```diff
@@ -0,0 +1,92 @@
+==============================
+Choose report page orientation
+==============================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Freporting--engine-lightgray.png?logo=github
+    :target: https://github.com/OCA/reporting-engine/tree/15.0/report_page_orientation
+    :alt: OCA/reporting-engine
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/reporting-engine-15-0/reporting-engine-15-0-report_page_orientation
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/143/15.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows the redesign of pdf reports, mixing portrait
+and landscape page appearance by defining rotation in a
+`<div>` element's class attribute.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+As a developer, adding *div* defined with `class='rotated-page-orientation'` will lead to pages
+with landscape appearance, whereas the undefined will have a portrait
+appearance. See below example::
+
+    <t t-foreach="range(100)" t-as="index">
+        <div class="rotated-page-orientation">
+             <p>
+                Hello these are on rotated pages <t t-esc="index"/>
+             </p>
+        </div>
+    </t>
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/reporting-engine/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/reporting-engine/issues/new?body=module:%20report_page_orientation%0Aversion:%2015.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* Sunflower IT
+
+Contributors
+~~~~~~~~~~~~
+
+* Giovanni Capalbo <giovanni@therp.nl>
```

Comment:
@thomaspaulb i should not be first author. 
made core idea , but the "placeholder" substitution, the wkhtml2pdf improvements, AND most importantly the the JS subst solution  are yours. keep alphabetical IMO.

Reply:
@KKamaa Could you fix this and add the both of us too, and then squash into a single commit? After that it is ready for merge.

Reply:
@thomaspaulb added the contributors though kept it alphabetical and squashed to one commit as you mentioned